cloudformation/slack-router-cfn-template.js

```diff
@@ -0,0 +1,94 @@
+const cf = require('@mapbox/cloudfriend')
+
+const Resources = {
+  SlackRouterApi: {
+    Type: 'AWS::ApiGatewayV2::Api',
+    Properties: {
+      Name: 'slack-router-api',
+      ProtocolType: 'HTTP',
+    },
+  },
+  SlackToLambdaIntegration: {
+    Type: 'AWS::ApiGatewayV2::Integration',
+    Properties: {
+      ApiId: cf.ref('SlackRouterApi'),
+      Description: 'Integration for Slack Slash Commands',
+      IntegrationType: 'AWS_PROXY',
+      IntegrationUri: cf.join('', [
```

Comment:
Can you replace this with a `cf.arn`?

Reply:
For the IntegrationURI? I followed the [docs about uri](https://docs.aws.amazon.com/apigateway/api-reference/resource/integration/#uri) but couldn't find an easier/cleaner way to do it. This honestly took me awhile to figure out so would appreciate any help.

@dakotabenjamin do you know?

Reply:
Ah, you are right. I can't seem to find a easier way to get the lambda URI. You could combine it with the definition for [`cf.arn`](https://github.com/mapbox/cloudfriend/blob/master/lib/intrinsic.js#L218-L222) if you wish, but this seems to be the right way to do it right now.